tests/main/views/test_suppliers.py

```diff
@@ -763,26 +763,16 @@ class TestSetSupplierDeclarations(BaseApplicationTest, FixtureMixin, JSONUpdateT
 
     def setup(self):
         super(TestSetSupplierDeclarations, self).setup()
-        with self.app.app_context():
-            framework = Framework(
-                slug='test-open',
-                name='Test open',
-                framework='g-cloud',
-                status='open')
-            db.session.add(framework)
-            db.session.commit()
+        framework = Framework(
+            id=100,
+            slug='test-open',
+            name='Test open',
+            framework='g-cloud',
+            status='open')
+        db.session.add(framework)
+        db.session.commit()
         self.setup_dummy_suppliers(1)
 
```

Comment:
Set the `id` to >= 100 so it it deleted by the `BaseApplicationTest.teardown` and we don't need to do it manually here.